### PR TITLE
Dependabot issue creation

### PR DESCRIPTION
**Pull request by mitchdawson1982**

### CODE REVIEW

File: .github/workflows/dependabot-create-issue.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+            if: github.event.pull_request.user.login  == 'dependabot[bot]'
+            env:
+              pr_title: ${{github.event.pull_request.title}}
+              pr_number: ${{github.event.pull_request.number}}
+              pr_url: ${{github.event.pull_request.url}}
+            run: |
+              project="Data Catalogue"
+              label="Dependabot Issue"
+              title="Dependabot PR $pr_title opened"
+              body="Dependabot has opened PR #$pr_number url - $pr_url
```
Comment (MatMoore) [inline .github/workflows/dependabot-create-issue.yml:26]: I think github might collapse #pr_number and #pr_url to the same thing, so you might not need both?

File: .github/workflows/dependabot-create-issue.yml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+          - name: Open issue if Dependabot PR
+            if: github.event.pull_request.user.login  == 'dependabot[bot]'
+            env:
+              pr_title: ${{github.event.pull_request.title}}
+              pr_number: ${{github.event.pull_request.number}}
+              pr_url: ${{github.event.pull_request.url}}
+            run: |
+              project="Data Catalogue"
+              label="Dependabot Issue"
+              title="Dependabot PR $pr_title opened"
```
Comment (MatMoore) [inline .github/workflows/dependabot-create-issue.yml:25]: Can we name this something like "Review dependabot PR: $pr_title" so it reflects the task that needs doing?

Also, where's the emoji? 🧌

**Review by MatMoore — APPROVED** [review]
:dependabot: 👍🏻

File: .github/workflows/dependabot-create-issue.yml (hunk 1 above)
Comment (mitchdawson1982) [reply .github/workflows/dependabot-create-issue.yml:28]: Looking it [this](https://docs.github.com/en/webhooks/webhook-events-and-payloads#pull_request) they still seem to be different things -

Comment (MatMoore) [reply .github/workflows/dependabot-create-issue.yml:28]: I mean when it renders the text, I can type `#802` or `https://github.com/ministryofjustice/find-moj-data/pull/802` and they both get rendered as the same link, #802

File: .github/workflows/dependabot-create-issue.yml (hunk 2 above)
Comment (mitchdawson1982) [reply .github/workflows/dependabot-create-issue.yml:27]: Updated and Emoji Added :-)

**Review by MatMoore — APPROVED** [review]

**Pull request merged**